pallets/xcm-weight-trader/src/lib.rs

```diff
@@ -150,6 +156,31 @@ pub mod pallet {
 		SupportedAssetRemoved { location: Location },
 	}
 
+	#[pallet::genesis_config]
+	pub struct GenesisConfig<T: Config> {
+		pub assets: Vec<XcmWeightTraderAssetInfo>,
+		pub _phantom: PhantomData<T>,
+	}
+
+	impl<T: Config> Default for GenesisConfig<T> {
+		fn default() -> Self {
+			Self {
+				assets: vec![],
+				_phantom: Default::default(),
+			}
+		}
+	}
+
+	#[pallet::genesis_build]
+	impl<T: Config> BuildGenesisConfig for GenesisConfig<T> {
+		fn build(&self) {
+			for asset in self.assets.clone() {
+				Pallet::<T>::do_add_asset(asset.location, asset.relative_price)
+					.expect("couldn't add asset");
```

Comment:
The comment above may apply also here.

Reply:
This is only used when building the chain spec, not on a running node. The error seems fine to me.